### DISCUSSION TITLE
onError, get error status

### DISCUSSION THREAD

**Discussion by Jayllyz** [honojs/hono#2435] [Q&A] [answered]
Hello, I'm having an issue retrieving the error status with the onError handler. Thanks for your help.
![image](https://github.com/honojs/hono/assets/16305216/94beed31-0d63-4a5b-942a-433382444a4d)

**Comment by NicoPlyley** [comment, marked as answer]
There is no status, it's based off web standards which you can see [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/error). Some errors in other libraries do include a status so you can optionally chain it for the cases that do. I also have `hono-error-handler` which can make error handling a lot easier

**Reply by Jayllyz** [reply]
Thank you for your answer.
